Sources/SWBCore/Settings/Settings.swift

```diff
@@ -1576,6 +1580,52 @@ private class SettingsBuilder {
             }
         }
 
+        for artifactBundle in artifactBundleInfo ?? [] {
+            pushTable(.exported) { table in
+                for artifact in artifactBundle.metadata.artifacts.values {
+                    switch artifact.type {
+                    case .staticLibrary:
+                        for variant in artifact.variants {
+                            guard let staticLibraryMetadata = variant.staticLibraryMetadata else {
+                                continue
+                            }
+                            var conditionSets: [MacroConditionSet?] = []
+                            if let triples = variant.supportedTriples {
+                                for triple in triples {
```

Comment:
Should the input triple here be normalized for comparison to the current triple?